docs/usage/security-and-permissions.md

```diff
@@ -7,10 +7,12 @@ This page talks about our security stance, and explains what permissions are nee
 Renovate is open source software, and comes with no guarantees or warranties of any kind.
 That said, we will try to fix security problems in a reasonable timeframe if possible.
 
-### No certifications
+### Certifications
 
-Renovate the Open Source project does **not** have ISO 27001 or SOC2 certifications.
-Mend the company, which maintains Renovate and operates the hosted Renovate App for GitHub, does have ISO 27001 and SOC2 certifications.
+Renovate the open source project is **not** certified.
+
+The company which maintains Renovate and provides the hosted Renovate App for GitHub is called Mend.
+Mend is ISO 27001 and SOC2 certified.
```

Comment:
Does the Mend website maybe have proof of the ISO 27001 and SOC2 certification? Or maybe there's a ISO 27001 and SOC2 register, that we can link to?

I found a mention of ISO 27001 in the Mend FAQ, but the section doesn't have an anchor link. So I can't link it in our docs.

![Mend FAQ certification](https://user-images.githubusercontent.com/34918129/177137992-9aba2021-307e-4a43-a506-461f8fe8ef2d.png)

Reply:
I don't know if it's common for companies to post proof online. I just thought if we had such proof online, it would be nice to link to the proof from our docs.

Reply:
I prefer to keep the status quo here